cost-analyzer/templates/_helpers.tpl

```diff
@@ -126,6 +143,13 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 app: cost-analyzer
 {{- end -}}
+{{- define "query-service.commonLabels" -}}
+app.kubernetes.io/name: query-service
+helm.sh/chart: {{ include "cost-analyzer.chart" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
include the query-service.chartLables here to avoid duplication

Reply:
Fixed.